aptos-move/aptos-validator-interface/src/lib.rs

```diff
@@ -46,6 +58,23 @@ pub trait AptosValidatorInterface: Sync {
         limit: u64,
     ) -> Result<(Vec<Transaction>, Vec<TransactionInfo>)>;
 
+    async fn get_and_filter_committed_transactions(
```

Comment:
As suggested previously, I would recommend using `get_committed_transactions` to implement this method here so that you can get automatic support for db instance as well.

Reply:
well you can do something like
```rust
  .get_commited_txns()?.zip().enumerate().map(|idx, (txn, txn_info)| {
      let version = begin_version + version;
      (versino, txn, txn_info)
})
```